src/commands/autocomplete/create.ts

```diff
@@ -8,7 +9,8 @@ import {AutocompleteBase} from '../../base'
 type CommandCompletion = {
   id: string
   description: string
-  flags: any
+  flags: any,
+  args: Parser.args.IArg[]
```

Comment:
If I am not mistaken, the type should be `Command.Arg[]` from `import {Command} from '@oclif/config'` instead.

It might not change much as the interfaces should be very close, however for the sake of maintainability that could be nice.